osbs/cli/main.py

```diff
@@ -701,6 +720,52 @@ def cli():
                               dest="compose_ids", type=int, help="ODCS compose"
                               "used, may be used multiple times")
 
+    build_source_container_parser = subparsers.add_parser(
+        str_on_2_unicode_on_3('build-source-container'),
+        help='build a source container image in OSBS'
+    )
+    build_source_container_parser.add_argument(
+        "--sources-for-koji-build-nvr", action='store', required=True,
```

Comment:
what about 'sources_for_koji_build_id' ? koji-c will get both nvr & id if provided, so we should get both